stdlib/3/unittest/__init__.pyi

```diff
@@ -298,22 +302,30 @@ _TextTestResult = TextTestResult
 
 defaultTestLoader = ...  # type: TestLoader
 
-_ResultClassType = Callable[[TextIO, bool, int], TestResult]
+_ResultClassType = Callable[[IO[str], bool, int], TestResult]
 
 class TestRunner:
     def run(self, test: Union[TestSuite, TestCase]) -> TestResult: ...
 
 class TextTestRunner(TestRunner):
```

Comment:
The source code of cpython is here.
https://github.com/python/cpython/blob/0a16bb15afec28f355bc28203b6b10610293f026/Lib/unittest/runner.py#L120-L221